src/checkboxes/template.njk

```diff
@@ -20,6 +30,11 @@
         for: id
       }) | indent(4) | trim }}
     </div>
+    {% if item.conditional %}
+      <div class="govuk-checkboxes__conditional" id="{{ conditionalId }}">
+        {{ item.conditional.html | safe }}
```

Comment:
This feels like a big security risk considering we expect people to put html in here that contains form inputs which often replay user input...

Reply:
see https://github.com/alphagov/govuk-frontend/issues/514